ApplicationView/MenuRow/ApplicationMenuRow.cs

```diff
@@ -69,7 +69,8 @@ public ApplicationMenuRow()
 			this.AddChild(new MenuOptionSettings());
 
 			// put in the help menu
-			if (ActiveSliceSettings.Instance?.ActionMacros().Any() != null)
+			if (ActiveSliceSettings.Instance != null 
+				&& ActiveSliceSettings.Instance.ActionMacros().Any())
 			{
```

Comment:
I did not investigate thoroughly but the new code was not producing the same result. The menu was showing when no macro commands. Did it need to be != false?